test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_core_test.clj

```diff
@@ -207,9 +205,13 @@
       (update-in [connector-keyword :so-linger-milliseconds] identity)
       (update-in [connector-keyword :idle-timeout-milliseconds] identity)))
 
+
+(def default-http-connector-config
+  {:jmx-enable "false"})
```

Comment:
Why was this (and the change on line 214) necessary? (If I change 214 back to just `config` everything still seems to work.) Also, it seems kind of odd to call this `default-http-connector-config` if the default is set to true in our code... but I feel like I am probably missing what exactly this is used for?